pom.xml

```diff
@@ -74,7 +74,7 @@
 
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
 
-    <sonar.version>6.1</sonar.version>
+    <sonar.version>6.2-RC2</sonar.version>
```

Comment:
A bit sad that we have to rely on RC2 for the time being. Hopefully, 6.2 will be released before release of Java 4.4 and we will be able to update the dependency to the released version.